### ISSUE TITLE
Great idea

### ISSUE THREAD

**Issue by pimterry** [getsentry/osspledge.com#2]
Hi,

I just ran into this project after asking around on Twitter looking for exactly this (https://x.com/pimterry/status/1805948814831849718).

I don't work for Sentry, but I'm very much interested in concepts like this and helping sustain open source in general (I run [a profitable & 100% open-source indie company](https://httptoolkit.com), I maintain a bunch of popular JS community packages, and I'm a Node.js core collaborator).

Really just wanted to pop up to say that I think this is a great idea! From the outside it's not clear what the current state & todo list is, or if there's any way for outsiders to get directly involved (is this intended to be a Sentry-only project or a broader community thing that Sentry backs?), but if there's anything practical I can do to help I'd love to hear about it.

**Comment by vladh** [comment]
Just wanted to chime in to +1 everything @pimterry is saying, including the offer to help, and the suggestion that the model [1% for the Planet](https://www.onepercentfortheplanet.org/) are using might be of great use.

**Comment by chadwhitacre** [comment]
Thank you both for jumping in! Now that this repo exists, I've transferred [our primary ticket](https://github.com/getsentry/osspledge.com/issues/4) here, we can use that for high-level coordination. I've also added [onepercentfortheplanet.org](https://www.onepercentfortheplanet.org/) to the list of inspirations there. Hopefully that ticket can start to provide some context. Now that we're getting more visibility we should start filling out the description there and probably also the README.

P.S. @pimterry TIL HTTP Toolkit, so glad to hear you're finding success and sustainability!

Gonna close this here, see you over on https://github.com/getsentry/osspledge.com/issues/4 and beyond! 🙏 🙌 💃
